PF/Assignment2/PascalTringleTest.java

```diff
@@ -0,0 +1,51 @@
+import static org.junit.Assert.*;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PascalTringleTest {
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void testSpaces() {
+		assertEquals("    ", PascalTringle.spaces(1, 5));
```

Comment:
ACTION: to check for number of spaces, it is better to check the length of the String as it will be easy to understand. E.g. assertEquals(4, PascalTringle.spaces(1, 5).length())